src/en/administration/from_scratch.md

```diff
@@ -118,7 +120,7 @@ yarn build:prod
 exit
 ```
 
-Add another systemd unit file, this time for lemmy-ui. You need to replace example.com with your actual domain. Put the file in `/etc/systemd/system/lemmy-ui.service`, then run `systemctl enable lemmy-ui` and `systemctl start lemmy-ui`.
+Add another systemd unit file, this time for lemmy-ui. You need to replace example.com with your actual domain. Put the file in `/etc/systemd/system/lemmy-ui.service`, then run `systemctl enable lemmy-ui` and `systemctl start lemmy-ui`. Note that some of the environment variables `LEMMY_*` are subject to change, and you should use the most up to date variable names, as [found here](https://github.com/LemmyNet/lemmy-ui#configuration)
```

Comment:
I dont think there are any plans to change the env vars again, so remove this line.

Reply:
Ah good. I can get rid of the "Note that the..." part, but should I still keep the link to the config page?

Reply:
A note regarding environment variables, the 'lemmy' service part of these 'From Scratch' docs does not set a RUST_LOG default value. The Docker install does set a default value...

Reply:
I kept the config link, but removed the part where I said the envs might change. I also added a rust log.